src/Microsoft.PowerShell.Commands.Utility/commands/utility/GetDateCommand.cs

```diff
@@ -567,6 +568,30 @@ private string UFormatDateString(DateTime dateTime)
             return StringUtil.Format(sb.ToString(), dateTime);
         }
 
+        /// <summary>
+        /// This method returns the year of the ISO week calendar, which assigns each entire week to the Gregorian year that contains Thursday.
+        /// </summary>
+        private static int GetIsoYear(DateTime dateTime)
```

Comment:
Can we use ISOWeek.GetYear() ?
https://source.dot.net/#System.Private.CoreLib/ISOWeek.cs,37

Reply:
I'm happy to try it! I'd hate to be reinventing a wheel.

Reply:
Your link was extremely helpful. Without yet having a functioning build environment locally, I couldn't locate the ISOWeek object. I'll have to submit a separate PR to switch `%u` to use `ISOWeek.GetWeekday()`, instead of my reimplementation of it.